playbooks/metricsaggregation.yaml

```diff
@@ -3,4 +3,7 @@
   vars_files:
     - _defaults.yml
   roles:
+  - role: KubevirtCircuitBreaker
+    vars:
+    - cr_kind: KubevirtMetricsAggregation
```

Comment:
I got a use case for this while working on https://github.com/MarSik/kubevirt-ssp-operator/pull/98